proj/hps_accel/vizier_autotune.ipynb

```diff
@@ -0,0 +1,295 @@
+{
```

Comment:
you may want to consider https://jupytext.readthedocs.io/, if you want to maintain this as plain python script.